packages/airnode-validator/src/config/config.ts

```diff
@@ -52,7 +52,7 @@ export const providerSchema = z
 
 export const amountSchema = z
   .object({
-    value: z.number(),
+    value: z.number().lte(9007199254740991), // 2**53 - 1
```

Comment:
Do we have this problem only with the amount field?

Reply:
This would be relevant theoretically anywhere we might anticipate the user hardcodes a very very large **number**. My motivation here (though it is still probably unlikely) is protecting against a user denominating in `wei` instead of e.g. `ether` for an amount. 

Looking through `config.json` I think we're safe otherwise; the only other instance where I initially thought this might be relevant was the `_times` reserved parameter, but we're safe because that is a [string and not a number](https://github.com/api3dao/airnode/blob/bd4becb68ba334958b598f5a56e0e31278b0a71d/packages/airnode-validator/src/ois/ois.ts#L64).

I also looked through our codebase and couldn't find any instances where we manually specify overly large numbers (by searching for `000_000`, `2**`, `10**`, `.pow` ).